pyseir/inference/infer_rt.py

```diff
@@ -578,13 +723,14 @@ def infer_all(self, plot=True, shift_deaths=0):
                 )
 
             if "Rt_ci5__new_cases" in df_all:
-                plt.fill_between(
-                    df_all.index,
-                    df_all["Rt_ci5__new_cases"],
-                    df_all["Rt_ci95__new_cases"],
-                    alpha=0.2,
-                    color="steelblue",
-                )
+                if not InferRtConstants.DISABLE_DEATHS:
```

Comment:
I just plot the confidence interval on the composite Rt value - that's why I removed the one on cases. I could leave the case one there for comparison so that the effect of smoothing Rt on narrowing confidence intervals would be more obvious.